### ISSUE TITLE
SVG export error

### ISSUE THREAD

**Issue by webhazrat** [highcharts/highcharts#16589]
When export this chart by svg format an error occur. 
Error Like: 
This page contains the following errors:
error on line 1 at column 284: Extra content at the end of the document
Below is a rendering of the page up to the first error.

**Comment by weathda** [comment]
We have also experienced this issue with download SVG images from Highchart (v9.1.2).

![image](https://user-images.githubusercontent.com/55594498/144279059-66a3b3bc-7340-4a2e-aa61-904cc710b5c2.png)

Opening the image with Edge or Chrome results in the same error. 

![image](https://user-images.githubusercontent.com/55594498/144279208-81a6061d-5cf2-44c0-a2d8-031ef3d575a2.png)

**Comment by weathda** [comment]
Retested with 9.3.2 and still seeing the issue. Attached is the chart file. Remove .txt to get the original svg file.


[chart.svg.txt](https://github.com/highcharts/highcharts/files/7636826/chart.svg.txt)

**Comment by raf18seb** [comment]
Hi @weathda, thank you for providing more info.

I was able to reproduce the issue with an `accessibility.js` module.
Demo: https://jsfiddle.net/BlackLabel/y2Lpo3jr/

Seems like it's caused by the proxy elements added by this commit: https://github.com/highcharts/highcharts/commit/c2d291237af08b68c0c29263f54dfb815034a902

However, I wasn't able to reproduce the issue in v9.1.2.
It's working fine in v9.2.2, broken since v9.3.0 (current version is v9.3.2), but maybe I've missed something?

**Comment by weathda** [comment]
Oops, looks like my older svg was generated with 9.3.2 so I guess I had a newer version.

Thanks, I'll look for the fix in a newer release.

**Comment by oysteinmoseng** [comment]
Here is a drop-in workaround for this issue for those of you waiting for the next release: https://jsfiddle.net/oysteinmoseng/yezu6nx3/
